api-specificatie/openapi.yaml

```diff
@@ -81,7 +81,7 @@ paths:
           content:
             application/hal+json:
               schema:
-                $ref: '#/components/schemas/ZoekSuggestieResponse'
+                $ref: '#/components/schemas/KeuzelijstCollectieHal'
```

Comment:
Waarom heet deze KeuzelijstCollectieHal en niet gewoon Keuzelijst? Er zitten geen hal links o.i.d. in maar alleen een array met keuze items. Een keuze item bevat  een adres string, ADO ID en NUM ID.

Reply:
Ik vermoed dat we wel links op willen nemen naar de nummeraanduiding, adres en adresseerbaarobject. Geen embedded overigens hier. Ik heb #43 aangemaakt om dee discussie te borgen